04week/forEach.js

```diff
@@ -0,0 +1,12 @@
+
+
+
+function forEach(array, fn){
+  for(var i = 0; i < array.length; i++) {
+    fn(array[i])
```

Comment:
this should be also calling the function with the current index and the original array parameters